plugins/view-resources/src/components/UserStatus.svelte

```diff
@@ -0,0 +1,85 @@
+<!--
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import { createQuery } from '@hcengineering/presentation'
+  import core, { Account, Ref, UserStatus } from '@hcengineering/core'
+
+  export let user: Ref<Account>
+  export let size: 'small' | 'medium' = 'small'
+  export let background: string = 'red'
+
+  const query = createQuery()
+
+  let status: UserStatus | undefined = undefined
+
+  $: query.query(
```

Comment:
Just an idea, maybe put statuses into svelte store so we do not query them?

Reply:
Done.